taxonomy-colleges.php

```diff
@@ -0,0 +1,169 @@
+<?php
+	get_header();
+	$term = get_queried_object();
+?>
+<article role="main">
+	<section class="section-lead">
+		<div class="container my-5">
+			<div class="row">
+				<div class="col-sm-12 col-md-8 lead">
+					<?php echo get_field( 'college_page_lead_copy', 'colleges_' . $term->term_id ); ?>
+				</div>
+				<div class="col-sm-12 col-md-4">
+					<img src="<?php echo get_field( 'college_page_lead_image', 'colleges_' . $term->term_id )['url']; ?>" alt="" class="img-fluid">
```

Comment:
You should probably make sure this is set before displaying it. There is the possibility of displaying an image with an empty source here.